### ISSUE TITLE
Small suggestion

### ISSUE THREAD

**Issue by DavidDecraene** [sparticle999/SpaceCompany#323]
As a small quality of life improvement I would suggest to make the middle panel, with the machine options, scrollable.

For example, when Looking at energy options I need to scroll down until the energy and other resources are out of view before I can purchase the lava based and other generators, would be nice to still be able to see total values of everything available. Allowing a scroll would to that (max height + overflow-y: scroll)

**Comment by darkniko** [comment]
Yeah u are absolutely right. It is very good idea. Unfortunately if i recall correctly i believe this was already suggested and i think sparticle said something about the program or the engine used for the game didnt allow "formatting"  or it was kind of hard to make it possible. Idk ... sth like that ... But yes thats would be awesome....

**Comment by DavidDecraene** [comment]
Such a thing is trivial from a CSS perspective

#energyTab .container { max-height: 900px; overflow-y: scroll }

better or more refined approaches may exist, but this already gives scrollability.
